cluster/src/main/java/org/apache/iotdb/cluster/RemoteTsFileResource.java

```diff
@@ -47,7 +47,7 @@
 
   public RemoteTsFileResource() {
     setClosed(true);
-    this.timeIndex = IoTDBDescriptor.getInstance().getConfig().getTimeIndexLevel().getTimeIndex();
+    this.timeIndex = new DeviceTimeIndex();
```

Comment:
hi , Thanks your contribution.
I have a minor question .  why does delete TimeIndexlevel ? could you please write some reason? or 
first , create a jira issue or github issue for desc your purpose.
and then, raise pr link this issue. 
this way for understands.